Dockerfile

```diff
@@ -1,12 +1,12 @@
-FROM codeclimate/alpine-ruby:b38
+FROM ruby:2.3-alpine
```

Comment:
One thing we're conscious of is the size of the images. What does this do to the size of this engine's Docker image?